src/bsp/include/bsp/IHostUart.h

```diff
@@ -1,12 +1,14 @@
 #ifndef __IHOSTUART_H__
 #define __IHOSTUART_H__
 
+#include <common/IProtobufStream.h>
 #include <cstdint>
 
-#define HOST_UART_MAX_MESSAGE_LENGTH UINT16_MAX
+#define HOST_UART_MAX_MESSAGE_LENGTH 1024
```

Comment:
Should we document these lengths for when this is used accross platforms? (such as android app)

Reply:
Added to confluence